functions/FunctionsExample/Shared/View/AddNumbersView.swift~

```diff
@@ -0,0 +1,94 @@
+//
```

Comment:
Seems like this file name has a bad extension (`swift~`), which causes code highlighting to not show up in the review UI.